modules/installation/pages/pubcloud-requirements.adoc

```diff
@@ -0,0 +1,80 @@
+[[public-cloud-requirements]]
+= Public Cloud Requirements
+
+You can run {productname} Server on a cloud instance from a third-party provider such as Amazon EC2, or Microsoft Azure.
+
+This section details the requirements for using {productname} on a public cloud instance.
+
+[NOTE]
+====
+Public clouds provide {productname} under a Bring Your Own Subscription (BYOS) model.
+This means that you must register instances with the {scc}.
+For more information about registering {productname} with {scc}, see xref:installation:general-requirements.adoc[].
+====
+
+Depending on the public cloud framework you are using, you can locate the {productname} images by searching for the  keywords [package]``suse``, [package]``manager``, [package]``proxy``, or [package]``BYOS``.
+
+
+== Instance Requirements
+
+Select a public cloud instance type that meets the hardware requirements in xref:installation:hardware-requirements.adoc[].
+
+Before you begin, here are some other considerations:
+
+* The {productname} setup procedure performs a forward-confirmed reverse DNS lookup.
+This must succeed in order for the setup procedure to complete and for {productname} to operate as expected.
+It is important to perform hostname and IP configuration before you set up {productname}.
+* {productname} Server and Proxy instances need to run in a network configuration that provides you control over DNS entries, but cannot be accessed from the internet at large.
+* Within this network configuration DNS resolution must be provided: `hostname -f` must return the fully-qualified domain name (FQDN).
+* DNS resolution is also important for connecting clients.
+* DNS is dependent on the cloud framework you choose, refer to the cloud service provider documentation for detailed instructions.
+* We recommend that you locate software repositories, the server database, and the proxy squid cache on an external virtual disk.
+This prevents data loss if the instance is unexpectedly terminated.
+This section includes instructions for setting up an external virtual disk.
+
+== Network Requirements
+
+On a public cloud service, you must run {productname} within a restricted network, such as VPC private subnet with an appropriate firewall setting.
+The instance must only be able to be accessed by machines in your specified IP ranges.
+
+[WARNING]
+====
+A world-accessible {productname} instance violates the terms of the {productname} EULA, and is not supported by {suse}.
+====
+
+To access the {productname} {webui}, allow HTTPS when configuring the network access controls.
+
+
+
+== Separate Storage Volumes
+
+We recommend that the repositories and the database for {productname} are stored on a virtual storage device.
```

Comment:
I think the important issue here is that the storage devie to separate from the SUMA partitions (from the OS); line 55 and 60 seem to indicate this. Everything is virtual here...